components/eam/src/chemistry/yaml/mo_drydep/drydep_inti_xactive_input_ts_0.yaml

```diff
@@ -0,0 +1,3 @@
00, -0.10180517035673147E+001, -0.87952191739420205E+000, -0.10724153591027763E+001, -0.11647151510034477E+001, -0.12617568795285550E+001, -0.12617568795285550E+001, -0.14321972701768837E+001, -0.12617568795285550E+001, -0.11647151510034477E+001, -0.14321972701768837E+001, -0.12617568795285550E+001, -0.10180517035673147E+001, -0.85017381732928443E+000, -0.10724153591027763E+001, -0.87952191739420205E+000, -0.87952191739420216E+000, -0.10724153591027763E+001, -0.85017381732928432E+000, -0.10180517035673147E+001, -0.12617568795285550E+001, -0.14321972701768837E+001, -0.11647151510034477E+001, -0.12617568795285550E+001, -0.14321972701768837E+001, -0.12617568795285550E+001, -0.12617568795285550E+001, -0.11647151510034477E+001, -0.10724153591027763E+001, -0.87952191739420205E+000, -0.10180517035673147E+001, -0.85017381732928432E+000, -0.79129952552206684E+000, -0.91987191184397255E+000, -0.70497830836867037E+000, -0.79129952552206673E+000, -0.10180517035673147E+001, -0.10724153591027763E+001, -0.85017381732928443E+000, -0.87952191739420205E+000, -0.10724153591027763E+001, -0.10180517035673147E+001, -0.87952191739420205E+000, -0.85017381732928443E+000, -0.91987191184397255E+000, -0.79129952552206684E+000, -0.79129952552206684E+000, -0.70497830836867037E+000,  0.70497830836867037E+000,  0.79129952552206673E+000,  0.79129952552206673E+000,  0.91987191184397255E+000,  0.85017381732928443E+000,  0.87952191739420216E+000,  0.10180517035673147E+001,  0.10724153591027763E+001,  0.87952191739420205E+000,  0.85017381732928443E+000,  0.10724153591027763E+001,  0.10180517035673147E+001,  0.79129952552206684E+000,  0.70497830836867037E+000,  0.91987191184397255E+000,  0.79129952552206673E+000,  0.85017381732928443E+000,  0.10180517035673147E+001,  0.87952191739420205E+000,  0.10724153591027763E+001,  0.11647151510034477E+001,  0.12617568795285550E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.11647151510034477E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.10180517035673147E+001,  0.85017381732928443E+000,  0.10724153591027763E+001,  0.87952191739420205E+000,  0.87952191739420216E+000,  0.10724153591027763E+001,  0.85017381732928432E+000,  0.10180517035673147E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.11647151510034477E+001,  0.12617568795285550E+001,  0.14321972701768837E+001,  0.12617568795285550E+001,  0.12617568795285550E+001,  0.11647151510034477E+001,  0.10724153591027763E+001,  0.87952191739420205E+000,  0.10180517035673147E+001,  0.85017381732928443E+000,  0.79129952552206684E+000,  0.91987191184397255E+000,  0.70497830836867026E+000,  0.79129952552206673E+000,  0.10180517035673147E+001,  0.10724153591027763E+001,  0.85017381732928443E+000,  0.87952191739420205E+000,  0.10724153591027763E+001,  0.10180517035673147E+001,  0.87952191739420205E+000,  0.85017381732928443E+000,  0.91987191184397255E+000,  0.79129952552206684E+000,  0.79129952552206684E+000,  0.70497830836867037E+000]
```

Comment:
We are missing the header of this yaml file:

```
mam4xx:
  function: drydep...
input:
```

Reply:
as function name, could you use `find_season_index`?

Reply:
This should be fixed now.